client/src/components/StarRating.jsx

```
@@ -0,0 +1,18 @@
+import React from 'react'
+import { assets } from '../assets/assets'
+
+const StarRating = ({ rating = 4 }) => {
+    return (
+        <>
+            {Array(5).fill('').map((_, index) => (
+                <img
+                    className='w-4.5 h-4.5'
+                    src={rating > index ? assets.starIconFilled : assets.starIconOutlined}
+                    alt="Rating"
+                />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)